forge-gui/res/cardsfolder/upcoming/rootcast_apprenticeship.txt

```diff
@@ -0,0 +1,10 @@
+Name:Rootcast Apprenticeship
+ManaCost:3 G
+Types:Sorcery
+A:SP$ Charm | Choices$ DBPutCounter,DBCopy,DBToken,DBSacrifice | CharmNum$ 3 | CanRepeatModes$ True
+SVar:DBPutCounter:DB$ PutCounter | ValidTgts$ Creature | CounterType$ P1P1 | CounterNum$ 3 | SpellDescription$ Put three +1/+1 counters on target creature.
+SVar:DBCopy:DB$ CopyPermanent | ValidTgts$ Permanent.token+YouCtrl | TgtPrompt$ Select target token you control | SpellDescription$ Create a token that's a copy of target token you control.
+SVar:DBToken:DB$ Token | ValidTgts$ Player | TokenAmount$ 1 | TokenScript$ g_1_1_squirrel | TokenOwner$ TargetedPlayer | StackDescription$ SpellDescription | SpellDescription$ Target player creates a 1/1 green Squirrel creature token.

```

Comment:
you screwed yourself there by overriding default TokenOwner since you're targeting multiple players

Reply:
OK, so what should I do about it?

Reply:
just remove it...